src/users/dto/update-user.dto.ts

```diff
@@ -0,0 +1,34 @@
+
+import { ApiProperty, PartialType } from '@nestjs/swagger';
+import { CreateUserDto } from './create-user.dto';
+import { UserRole } from '../../constants';
+import { IsEmail, IsEnum, IsNotEmpty, IsOptional, MinLength } from 'class-validator';
+import { isUniqueDb } from '@youba/nestjs-dbvalidator';
+import { Transform } from 'class-transformer';
+
+export class UpdateUserDto extends PartialType(CreateUserDto) {
+    @ApiProperty({ example: 'John' })
+    @IsNotEmpty()
+    @IsOptional()
```

Comment:
Estas 2 cosas no se contradicen xD?
```suggestion
    @IsNotEmpty()
    @IsOptional()
```

Reply:
Recontra corregido